app/src/features/search/SearchComponent.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Button, Input, InputAdornment, Theme } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { mdiMagnify } from '@mdi/js';
+import { Icon } from '@mdi/react';
+import { useFormikContext } from 'formik';
+import { IAdvancedSearch } from 'interfaces/useSearchApi.interface';
+import React from 'react';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  keywordSearch: {
+    height: '52px',
+    flex: '1 1 auto',
+    paddingLeft: theme.spacing(1.25),
+    display: 'flex',
+    alignItems: 'center',
+    border: '1px solid rgba(0, 0, 0, 0.23)',
+    borderRadius: '4px 0 0 4px',
+    backgroundColor: '#f6f6f6',
+    transition: 'all ease-out 0.25s',
+    '&:hover': {
+      borderColor: theme.palette.primary.main,
+      boxShadow: '0 0 0 1px #003366 inset'
+    },
+    '&:active': {
+      borderColor: theme.palette.primary.main,
+      boxShadow: '0 0 0 1px #003366 inset'
+    },
+    '&:focus': {
+      borderColor: theme.palette.primary.main,
+      boxShadow: '0 0 0 1px #003366 inset'
+    }
+  },
+  searchBtn: {
+    height: '100%',
+    minWidth: '8rem'
+  },
+  chipStyle: {
+    color: 'white',
+    backgroundColor: '#38598a',
+    textTransform: 'capitalize'
+  }
+}));
+
+export default () => {
```

Comment:
I think it would be best to assign this to a variable, and then export it, rather than exporting it as an anonymous function.
Partially for clarity, and partially to match the existing patterns.

Reply:
Oops! Good catch. Didn't intend to make this function anonymous